docs/platform/howto/project-support-center.rst

```diff
@@ -29,10 +74,70 @@ To create support tickets in the `Aiven Console <https://console.aiven.io/>`_:
     * Any error messages
     * Any languages or frameworks you are using
 
-#. Select the affected project, service, and a severity level.
+#. Optional: Upload files such as screenshots, logs, or :ref:`HAR files <create-har-files>`.
+   
+   .. important::
+        Aiven support will never ask you to provide sensitive data such as passwords or personal information. You should remove (not redact) any sensitive data in files that you attach to a support ticket.
```

Comment:
it kinda feels too short for such an important note, but can't think of anything to add either :) except maybe "You must remove...". should we explain any risks or that it may trigger a security incident on our side?